### ISSUE TITLE
Tabset: Add and open buttons don't visually indicate focus (2.4.7)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#867]
### Expected behavior

Each button visually indicates when it receives focus.
### Actual behavior

No visual indication on focus.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.12.0](https://github.com/blackbaud/skyux/releases/tag/v1.12.0)
